### DISCUSSION TITLE
Docker volume 2fa resets

### DISCUSSION THREAD

**Discussion by evoandroidevo** [authelia/authelia#5551] [Q&A] [answered]
I have noticed that when updating the container that 2fa resets for all users. I'm not sure if i have an option set wrong or something. Thanks for the help.

**Comment by james-d-elliott** [comment, marked as answer]
Sorry for not getting back to you sooner.

This will occur if you're using sqlite for the storage section and you have not adequately configured the volume you are trying to save it on. You can check the `path` value is correctly set to a path on the left side of a volume setting from this example here: https://www.authelia.com/configuration/storage/sqlite/#configuration

By default the `/config` dir is a volume.
